public/app.js

```diff
@@ -8,10 +8,10 @@ function updateStatus(currentValue, id) {
         if (currentValue.name.indexOf("BMA") > -1) {
             // bma
             if (vehicle.find('.status').hasClass('status--0') && currentValue.status != 0) {
-                status.removeClass('status--'+oldStatus).addClass('status--'+currentValue.status);
```

Comment:
When in BMA object oldStatus has not a valid value, because of the bell icon. So `status--1`will be added but `status--0`won't be remove, which makes the background red. On reset `status--1`won't be removed so background still being red. 

Failed on bfafdb0 while switching from `switchClass`to `removeClass`and `addClass`.